subsys/net/l2/wifi/Kconfig

```diff
@@ -27,3 +27,9 @@ config WIFI_MGMT_RAW_SCAN_RESULTS_ONLY
 	  to the application.
 
 endif # WIFI_MGMT_RAW_SCAN_RESULTS
+
+config WIFI_MGMT_SCAN_MODE_PASSIVE
```

Comment:
Why do we even need this? This doesn't save any memory, so, we can enable this always.